apps/api/src/app/workflows-v2/usecases/build-variable-schema/build-available-variable-schema.usecase.ts

```diff
@@ -40,18 +50,15 @@ export class BuildVariableSchemaUsecase {
               format: 'date-time',
               description: 'The last time the subscriber was online (optional)',
             },
-            data: {
-              type: 'object',
-              properties: {},
-              description: 'Additional data about the subscriber',
-              additionalProperties: true,
-            },
+            data: buildVariablesSchema(
+              subscriber && typeof subscriber === 'object' && 'data' in subscriber ? subscriber.data : {}
```

Comment:
i find this more type safe, let me know what you think
```suggestion
              subscriber && typeof subscriber === 'object' && !!subscriber.data ? subscriber.data : {}
```

Reply:
I got `Property 'data' does not exist on type 'object'.`